tsdoc-config/src/TSDocConfigFile.ts

```diff
@@ -167,6 +172,10 @@ export class TSDocConfigFile {
     return this._supportForTags;
   }
 
+  public get allowedHtmlTags(): ReadonlyArray<string> | undefined {
+    return this._specifiesAllowedHtmlTags ? Array.from(this._allowedHtmlTags) : undefined;
```

Comment:
Could `_specifiesAllowedHtmlTags` be eliminated by representing the "not specified" state as `this._allowedHtmlTags === undefined`?

Reply:
Yes. I added `specifiesAllowedHtmlTags` to try to keep with the convention in this file that collections are `readonly`. There's no hard reason for it, though